utils/documentation_tests.txt

```diff
@@ -101,3 +101,4 @@ src/transformers/models/wavlm/modeling_wavlm.py
 src/transformers/models/whisper/modeling_whisper.py
 src/transformers/models/whisper/modeling_tf_whisper.py
 src/transformers/models/yolos/modeling_yolos.py
+src/transformers/models/bart/configuration_bart.py
```

Comment:
Could you put this line in a sorted order within this file 🙏

Reply:
Sure!